tests/posix/mt/mt.cmake

```diff
@@ -33,6 +33,7 @@ include(${SRC_DIR}/../posix-helpers.cmake)
 
 setup()
 
-execute(${TEST_EXECUTABLE} ${ops})
+execute(${TEST_EXECUTABLE} ${ops} --gtest_filter=mt.open_close_create_unlink)
```

Comment:
`ctest -j n` doesn't help run the tests faster?
I think, if this is needed, than these should definitely be added with two `add_test` commands.